File: src/main/java/no/nav/familie/ks/oppslag/dokarkiv/client/domene/Bruker.java
package no.nav.familie.ks.oppslag.dokarkiv.client.domene;

import javax.validation.constraints.NotNull;

public class Bruker {
    private IdType idType;
    private String id;

    public String getId() {
        return id;
    }

    public IdType getIdType() {
        return idType;
    }

    public Bruker(@NotNull(message = "Bruker mangler idType") IdType idType, @NotNull(message = "Bruker mangler id") String id) {
        this.idType = idType;
        this.id = id;
    }
}

File: src/main/java/no/nav/familie/integrasjoner/dokarkiv/client/domene/Bruker.kt
package no.nav.familie.integrasjoner.dokarkiv.client.domene

import javax.validation.constraints.NotNull

class Bruker(val idType: @NotNull(message = "Bruker mangler idType") IdType?,
             val id: @NotNull(message = "Bruker mangler id") String?)